easybuild/tools/filetools.py

```diff
@@ -343,10 +344,15 @@ def download_file(filename, url, path, forced=False):
     downloaded = False
     max_attempts = 3
     attempt_cnt = 0
+
+    # use blank HTTP header
+    header={'User-Agent':'EasyBuild'}
```

Comment:
style nitpicking: space after `:` (and spaces around `=`, but irrelevant with comment below)